Dockerfile

```diff
@@ -0,0 +1,39 @@
+# TODO: select a base image
+# Tip: start with a full base image, and then see if you can optimize with
+#      a slim or tensorflow base
+
+#      Standard version
+FROM python:3.10.10-buster
+
+#      Slim version
+# FROM python:3.10.10-slim-buster
+
+#      Tensorflow version
+# FROM tensorflow/tensorflow:2.10.0
+
+#      Or tensorflow to run on Apple Silicon (M1 / M2)
+# FROM armswdev/tensorflow-arm-neoverse:r22.11-tf-2.10.0-eigen
+
+
+# Copy everything we need into the image
+COPY baseline_model baseline_model
+COPY api api
```

Comment:
api folder is not yet created, right?

Reply:
api folder created containing a fast.py file.